pages/docs/how-to-guides/events.mdx

```diff
@@ -20,7 +20,7 @@ All future events also appear on the Events page for the app.
 
 _If you need help on how to edit this file, please see the <Link href="/docs/how-to-guides/editing">Editing Guide</Link>_
 
-3. This `json` file will contain one object for the event and must have these six fields: `isVirtual` and/or `isInPerson`, `color`, `name`, `description`, `date`, `url`, and `location`. It can also optionally include other fields. For instance: `userStatus` & `speakingTopic`. It will look like this:
+3. This `json` file will contain one object for the event and must have these six fields: `isVirtual` and/or `isInPerson`, `color`, `name`, `description`, `date`, `url`, and `location` (this field will be required only when `isInPerson` is true). It can also optionally include other fields. For instance: `userStatus` & `speakingTopic`. It will look like this:
```

Comment:
```suggestion
3. This `json` file will contain one object for the event and must have these six fields: `isVirtual` and/or `isInPerson`, `color`, `name`, `description`, `date`, `url`, and `location` (this field will be required only when `isInPerson` is `true`). It can also optionally include other fields. For instance: `userStatus` & `speakingTopic`. It will look like this:
```